testing_exercise/testing.js

```diff
@@ -0,0 +1,62 @@
+function replaceWith(string, ltrOld, ltrNew) {
+	var newString = "";
+	for(var i=0; i<string.length; i++) {
+		if(string[i]==ltrOld) {
+			newString += ltrNew;
+		} else {
+			newString += string[i];
+		}
+	};
+	if(newString === "") {
+		return string;
+	};
+	return newString;
+}
+
+//
+
+function expand(arr, num) {
+	var newArr = [];
+	for(var i=0; i < arr.length*num; i++) {
+		newArr.push(arr[i % arr.length]);
+	}
+	return newArr;
+}
+
+//
+
+function acceptNumbersOnly(args) {
+	for(var i=0; i<arguments.length; i++) {
+		if(isNaN(arguments[i]))
```

Comment:
I prefer to use { } around if statements even if it's one line.  If you want to make it a little cleaner, I'm ok with this:

```
if (isNaN(arguments[i]) { return false; }
```